qa/L0_lifecycle/test.sh

```diff
@@ -450,7 +450,7 @@ done
 rm models/graphdef_float32_float32_float32/config.pbtxt
 
 SERVER_ARGS="--model-repository=`pwd`/models --model-repository=`pwd`/models_0 \
-             --exit-on-error=false --exit-timeout-secs=5"
+             --exit-on-error=false --exit-timeout-secs=5 --strict-model-config=true"
```

Comment:
Why auto-complete needs to be disabled for this test? If there is some credible reason behind it then we must add it as a comment on top of it.

Reply:
Added comment in the test as to why --strict-model-config=true was added.

Reply:
The test was run with the assumption that `--strict-model-config=true` (previous default behavior), so we need to explicitly set it if we move the default to false